vignettes/epinow.Rmd

```diff
@@ -30,17 +30,12 @@ This should be replaced with parameters relevant to the system that is being stu
 library("EpiNow2")
 options(mc.cores = 4)
 reported_cases <- example_confirmed[1:60]
-incubation_period <- get_incubation_period(
-  disease = "SARS-CoV-2", source = "lauer"
-)
 reporting_delay <- dist_spec(
   mean = convert_to_logmean(2, 1), mean_sd = 0,
   sd = convert_to_logsd(2, 1), sd_sd = 0, max = 10
 )
-delay <- incubation_period + reporting_delay
-generation_time <- get_generation_time(
-  disease = "SARS-CoV-2", source = "ganyani"
-)
+delay <- example_incubation_period + reporting_delay
+#> Error in eval(expr, envir, enclos): object 'example_incubation_period' not found
```

Comment:
it looks like lazy loading isn't working as expected here? Getting errors throughout :(

Reply:
That was totally my bad as I cunningly recompiled the vignette on a machine that had the version from main installed.